src/sentry/models/options/option.py

```diff
@@ -18,6 +33,9 @@ class BaseOption(Model):  # type: ignore
 
     key = models.CharField(max_length=128, unique=True)
     last_updated = models.DateTimeField(default=timezone.now)
+    last_updated_by = models.CharField(
+        max_length=128, choices=OptionsTypes.choices(), default="legacy"
```

Comment:
This `max_length` probably needs to be reflected in the migration state and the ddl